packages/kbn-lens-embeddable-utils/config_builder/charts/tag_cloud.ts

```diff
@@ -31,18 +30,17 @@ function getAccessorName(type: 'breakdown') {
 
 function buildVisualizationState(config: LensTagCloudConfig): TagcloudState {
   const layer = config;
-  const isFormula = isFormulaDataset(config.dataset) || isFormulaDataset(layer.dataset);
 
   return {
     layerId: DEFAULT_LAYER_ID,
-    valueAccessor: !isFormula ? layer.value : ACCESSOR,
+    valueAccessor: ACCESSOR,
     maxFontSize: 72,
     minFontSize: 12,
     orientation: 'single',
     showLabel: true,
     ...(layer.breakdown
       ? {
-          tagAccessor: !isFormula ? (layer.breakdown as string) : getAccessorName('breakdown'),
+          tagAccessor: getAccessorName('breakdown'),
```

Comment:
Why removing the formula support here and above?

Reply:
It doesnt remove the formula support. It is fixing a bug, if you check the rest implementations they use the same logic. The formula code is some lines below.